build/azure-pipeline.pre-release.yml

```diff
@@ -22,6 +22,8 @@ resources:
 extends:
   template: azure-pipelines/extension/pre-release.yml@templates
   parameters:
+    locTsConfigs: $(Build.SourcesDirectory)/tsconfig.json
+    locBundleDestination: $(Build.SourcesDirectory)/out
```

Comment:
As a reminder for future reviews of this PR, this should match the output `path:` in the webpack config: https://github.com/microsoft/vscode-jupyter/blob/main/build/webpack/webpack.extension.node.config.js#L142